client/webserver/site/src/js/markets.ts

```diff
@@ -2050,6 +2050,10 @@ export default class MarketsPage extends BasePage {
     mord.ord = order
     if (note.topic === 'MissedCancel') Doc.show(mord.details.cancelBttn)
     if (order.filled === order.qty) Doc.hide(mord.details.cancelBttn)
+    if (note.topic === 'OrderRetired') {
```

Comment:
@buck54321 How about in https://github.com/decred/dcrdex/pull/2171, you catch this order note topic to set a `canClear` flag for the order, only showing the sweep icon for such orders.  That would solve the reappearing order issue.

The alternative, which I'd really rather avoid, is to redefine the Your Orders to include more than what's in the `trades` map, because the intent there is to report only active (and very recently completed but not yet retired) orders.